### PR TITLE
docs(Banner): Remove outdated reference

### PR DESCRIPTION
**Pull request by yosefbennywidyo**
Removed the sentence referencing the "Announcement" component as the current page's "Usage" section already covers the relevant use cases for Banners.

### CODE REVIEW

**Comment by creative-media-ai** [comment]
Feedback

File: content/components/banner.mdx
Comment (creative-media-ai) [inline content/components/banner.mdx:1]: Feedback

**Comment by creative-media-ai** [comment]
Help or support

**Review by camertron — APPROVED** [review]

**Comment by camertron** [comment]
Thank you 👍

**Pull request merged**